qsiprep/workflows/dwi/derivatives.py

```diff
@@ -131,10 +131,14 @@ def init_dwi_derivatives_wf(source_file) -> Workflow:
             source_file=source_file,
             base_directory=output_dir,
             space="T1w",
+            statistic="cnr",
             desc=config.workflow.hmc_model,
```

Comment:
In qsirecon we use the [model entity](https://github.com/PennLINC/qsirecon/blob/38c3f082c0af4ecb5584507193cd8c79ce74afdb/qsirecon/tests/data/tortoise_recon_outputs.txt#L31), which I think would be better than desc